### DISCUSSION TITLE
Libraries bump

### DISCUSSION THREAD

**Discussion by danyhiol** [accounts-js/accounts#1246] [Q&A]
It would be nice to [bump some library](https://github.com/accounts-js/accounts/issues/1244), especially for the new apollo 4 which does have support for `GraphQl < 16.6.0`. I am using apollo in my app, and migrating to V4 is not possible as `@accounts/graphql-api` uses `GraphQL < 16`. 
I also realized that the auth directive in `@accounts/graphql-api` is not actual and does not work with new versions of `GraphQL`. 
It would be nice to review these else the `@accounts/graphql-api` package may not be usable anymore.

**Comment by darkbasic** [comment]
Tomorrow we will release the first release candidate of version 1.0 which will bump everything: https://github.com/accounts-js/accounts/pull/1258
